samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -104,7 +105,7 @@
   private TestRunner() {
     this.configs = new HashMap<>();
     this.inMemoryScope = RandomStringUtils.random(10, true, true);
-    configs.put(JobConfig.JOB_NAME(), JOB_NAME);
+    configs.put(ApplicationConfig.APP_NAME, JOB_NAME);
```

Comment:
Rename the `JOB_NAME`  constant to `APP_NAME`.

Reply:
Ok